### PR TITLE
Add card results

### PR DESCRIPTION
**Pull request by MilePaor**

### CODE REVIEW

**Comment by Niweera** [comment]
Hey @MilePaor, thank you for creating this pull request! Have a wonderful day...

File: src/components/SearchBox/index.js [hunk 1]
```diff
@@ -3,34 +3,56 @@ import React, { useState } from "react";
...
       />
-      {query.map(({ id, title }) => (
-        <div key={id}>
-          <SearchResultsTitle title={title} />
-        </div>
-      ))}
+      <div>
+        {results.map((x) => {
+          return (
+            <h4 id={x.id} onClick={() => setSelectedMovie(x)}>
```
Comment (Niweera) [inline src/components/SearchBox/index.js:35]: Please add a key to the element.

File: src/components/SearchBox/index.js [hunk 2]
```diff
@@ -3,34 +3,56 @@ import React, { useState } from "react";
...
+        })}
+      </div>
+
+      {selectedMovie && <SearchResultsTitle title={selectedMovie.title} />}
+
+      <div style={{ display: "flex" }}>
+        {selectedMovie &&
+          results.map((x) => {
+            return (
+              <MovieCard
```
Comment (Niweera) [inline src/components/SearchBox/index.js:48]: Please add a key to the element.

**Review by Niweera — CHANGES_REQUESTED** [review]
Please address the issues mentioned.

**Comment by Niweera** [comment]
Hi @MilePaor,
if the above issues are not addressed within the next 48 hrs, we will close this pull request.

**Comment by MilePaor** [comment]
@Niweera please check now.

**Review by Niweera — APPROVED** [review]

**Pull request merged**